recipe/meta.yaml

```diff
@@ -140,11 +150,12 @@ outputs:
       requires:
         - {{ compiler("c") }}
         - {{ compiler("cxx") }}
-        - gxx
-        - gcc
+        - gxx              # [linux]
+        - gcc              # [linux]
+        - vs{{ VSYEAR }}_win-64 =={{ CL_VERSION }}  # [win]
```

Comment:
Should we add to `run_constrained` with `vc` somehow? Maybe something like this above?

```YAML
        - vc >=14.2  # [win]
```